### ISSUE TITLE
Bug/Feature request: Skip non-existend users while downloading from list (like in solved #1011)

### ISSUE THREAD

**Issue by Napoli0n** [Nandaka/PixivUtil2#1325]
### Prerequisites

* [x] Did you read FAQ section in [readme.md](https://github.com/Nandaka/PixivUtil2/blob/master/readme.md)?
* [x] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?
* [x] Did you search for existing issues in [Issues](https://github.com/Nandaka/PixivUtil2/issues)?

### Description

Download from list breaks (stops after 4 retries) when the user is not found.
I guess it is the same error/problem as in issue report [#1011 ](https://github.com/Nandaka/PixivUtil2/issues/1011)

Error Message (partial extract):
> Member Url    : https://www.pixiv.net/ajax/user/81858474/profile/all
> Something wrong, retrying after 2 second (4) ==> PixivException(1003 User has left pixiv or the user ID does not exist., hasDumpPage=Y)
> 2 of 2s.
> Member Url    : https://www.pixiv.net/ajax/user/81858474/profile/all
> Giving up member_id: 81858474 when processing PixivSketch ==> PixivException(1003 User has left pixiv or the user ID does not exist., hasDumpPage=Y)
> done for member id = 81858474.
> 
> Unknown errors from previous operation
> Member: 81858474 ==> User has left pixiv or the user ID does not exist.

Then PixivUtil2 goes back to the context menu

### Steps to Reproduce

1. Add user 81858474 somewhere to a list and try to download.
2. PixivUtil2 downloads all entries up to the missing user.
3. Download stops after 4 tries of missing user with error message

**Expected behavior:** 

The download should skip the missing user and continue with the next user in the list.
Optional behavior: Show all skipped users + number of skipped users at the end to review.

### Versions
PixivDownloader2 version 20230105

**Comment by biggestsonicfan** [comment]
This is intended behavior. It should never simply skip a user because they don't exist, PixivUtil2 is trying to tell you there is a problem when there is a problem. The problem is the Pixiv user does not exist. Therefore, the problem lies with your list, not PixivUtil2. Remove the user from the list or double check to ensure you have the correct user ID.

**Comment by Napoli0n** [comment]
I disagree with your view but it's of course a reasonable workaround (for now).
My reasons for the suggested behavior are, that when I want to use a list, I expect the program to download all (available) users in it and not stop at the first problem if it could just have continued.
In my case, I went to sleep, expecting it would download 20 users but when I woke up, I noticed, that it had only done 4 before stopping because of the missing user.
The biggest problem is, that you can't be sure, when a user deletes its profile. This could even happen in the few seconds between creating a new list and starting the download, and there is no reliable way to prevent that.

You can still be notified about the errors at the end of all other downloads, as I suggested. Stopping in between seems not so useful. Especially as it is a catchable "error" and not a real problem within the program.

**Comment by biggestsonicfan** [comment]
I'm not sure if you know, but if you choose option 1, and put in a space delimited sentence `xxx yyy zzz` of users, it works like this:

```
Input: 1
Member ids: 81858474 33706539
Start Page (default=1):
End Page (default= 0, 0 for no limit):
Including Pixiv Sketch.
Found 2 ids
Member IDs: ['81858474', '33706539']
Processing Member Id: 81858474
Page 1 of 0
Member Url    : https://www.pixiv.net/ajax/user/81858474/profile/all
Member ID (81858474): PixivException(1003 User has left pixiv or the user ID does not exist., hasDumpPage=Y)
User has left pixiv or the user ID does not exist.
Member Url    : https://www.pixiv.net/ajax/user/81858474/profile/all
Member ID: 81858474 is not valid
Processing Member Id: 33706539
```

The error is there to inform you your list is bad so you can correct it. Otherwise, use option 1.

**Comment by biggestsonicfan** [comment]
Please disregard my previous posts. I think I have an answer. We both made assumptions and I think we were both wrong.

Pixivutil2 actually _sorts_ lists. If my list.txt file contains:
```
200
4
5
3
1
2
```
It will process them in the numerical order 1,2,3,4,5,200. 

I am now 99% sure that your member id `81858474` was the highest ID on your list if you sort it. Please verify the other member ids on your list that came after this member id and check the logs to see if they were processed.

**Comment by Napoli0n** [comment]
Thank you and sorry for the late response. You are correct. Pixivutil2 seems actually to react like I prefer.
I checked it now with shorter lists and different examples. 

I wasn't aware, that the list was sorted in the background and therefore when I checked the last view entries in my list against the last console outputs, it looked like it stopped before finishing. To top it off,  "81858474" was the highest number in my list, as you suspected, and therefore was processed last, confounding my confusion with the missing user error (from the middle of my list) right before it finished.
